www/src/pages/foundations/colors.tsx

```diff
@@ -57,30 +55,27 @@ export interface ISwatch {
   isUnused?: boolean,
 }
 
+const styles = typeof window !== 'undefined' ? getComputedStyle(document.body) : null;
```

Comment:
@PKulkoRaccoonGang can you please move this line inside `ColorsPage` component and pass `styles` as props to `Swatch`  component? I know theme switching works now and values get recalculated, but I'm just a little worried it might still cause bugs if we use `getComputedStyle(document.body)` outside of components.

Reply:
Done
I would also like to discuss with the @monteri  the removal of the style assembly script in development

Reply:
Thanks!